.github/workflows/ci.yml

```diff
@@ -398,5 +398,5 @@ jobs:
       - lint
       - rust_check
       - rust_test
-    if: ${{ (success() || cancelled()) && github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
why not main branch? 🤨

Reply:
Because the branches for `push` are already specified at the top of the workflow.

```yml
  push:
    branches:
      - main
    paths-ignore:
      - "**/*.md"
      - "website/**"
    tags-ignore:
      - "**"
```